libs/hwServer/constants.js

```diff
@@ -47,7 +47,7 @@ export const FUNCTION_TYPES = {
 };
 
 export const METHOD_NAMES = {
-  [IPC_MESSAGES.CHECK_LEDGER]: 'checkLedger',
+  [IPC_MESSAGES.CHECK_LEDGER]: 'checkStatus',
```

Comment:
@soroushm I believe the object key here should also be renamed to `IPC_MESSAGES.CHECK_STATUS`

Reply:
i just avoid to change it becouse of the conflict but as must the pr already merged its changed